CONTRIBUTING.md

```diff
@@ -180,6 +181,22 @@ base](https://github.com/mlflow/mlflow/blob/cdc6a651d5af0f29bd448d2c87a198cf5d32
 For more information about Plugins, see
 <https://mlflow.org/docs/latest/plugins.html>.
 
+### Python Style Guide
+##### Docstrings
+We follow [Google's Python Style Guide](https://google.github.io/styleguide/pyguide.html) for writing 
+docstrings. Make sure your docstrings adhere to this style guide.
+
+Currently, the MLflow team is working to reformat docstrings to 
+adhere to these guidelines. Any contributions regarding style 
+reformatting are much appreciated!
+
+###### Code Style
+We use [pylint](https://pypi.org/project/pylint/), 
+[black](https://black.readthedocs.io/en/stable/the_black_code_style/index.html), 
+and [ruff](https://github.com/astral-sh/ruff) in our CI via
+pre-commit Git hooks. If your code passes the CI checks, it's
+formatted correctly. 
+
```

Comment:
Can we mention how a contributor can validate that they have the correct version of black and ruff installed (as well as the doc-specific black extension versions) since the linting will be off if the local installations do not match what is running in CI?